web/beacon-app/src/hooks/useResendEmail.ts

```diff
@@ -39,12 +41,25 @@ const useResendEmail = () => {
       setLoading(false);
     }
   };
-
-  const reset = () => {
+  const reset = useCallback(() => {
     setLoading(false);
     setError(false);
     setResult(false);
-  };
+  }, []);
+
+  useEffect(() => {
+    if (result) {
+      // close other toast if any
+      toast.dismiss();
```

Comment:
Will the `SuccessfulAccountCreation` component need to be updated now that the hook includes the toast message? I believe that a user would see the toast message twice if they click the resend verification email button on the `verify-account` page.